drivers/npmx/npmx_shell.c

```diff
@@ -4109,616 +3617,637 @@ static int cmd_reset(const struct shell *shell, size_t argc, char **argv)
+					 "Buck active discharge", NULL),
+			       SHELL_CMD(gpio, &sub_buck_gpio, "Buck GPIO", NULL),
+			       SHELL_CMD(mode, NULL, "Set buck mode", cmd_buck_mode_set),
+			       SHELL_CMD(status, &sub_buck_status, "Buck status", NULL),
+			       SHELL_CMD(voltage, &sub_buck_voltage, "Buck voltage", NULL),
+			       SHELL_CMD(vout_select, &sub_buck_vout_select,
+					 "Buck output voltage reference source", NULL),
+			       SHELL_SUBCMD_SET_END);
+
+/* Creating subcommands (level 3 command) array for command "charger charging_current". */
+SHELL_STATIC_SUBCMD_SET_CREATE(sub_charger_charging_current,
+			       SHELL_CMD(set, NULL, "Set charging current",
+					 cmd_charger_charging_current_set),
+			       SHELL_CMD(get, NULL, "Get charging current",
+					 cmd_charger_charging_current_get),
 			       SHELL_SUBCMD_SET_END);
 
 /* Creating subcommands (level 4 command) array for command "charger die_temp resume". */
-SHELL_STATIC_SUBCMD_SET_CREATE(sub_die_temp_resume,
-			       SHELL_CMD(get, NULL,
-					 "Get die temperature threshold for resuming charging",
-					 cmd_die_temp_resume_get),
+SHELL_STATIC_SUBCMD_SET_CREATE(sub_charger_die_temp_resume,
 			       SHELL_CMD(set, NULL,
-					 "Set die temperature threshold for resuming charging",
-					 cmd_die_temp_resume_set),
+					 "Set die temperature threshold for resume charging",
+					 cmd_charger_die_temp_resume_set),
+			       SHELL_CMD(get, NULL,
+					 "Get die temperature threshold for resume charging",
+					 cmd_charger_die_temp_resume_get),
 			       SHELL_SUBCMD_SET_END);
 
 /* Creating subcommands (level 4 command) array for command "charger die_temp stop". */
-SHELL_STATIC_SUBCMD_SET_CREATE(sub_die_temp_stop,
-			       SHELL_CMD(get, NULL,
-					 "Get die temperature threshold for stop charging",
-					 cmd_die_temp_stop_get),
+SHELL_STATIC_SUBCMD_SET_CREATE(sub_charger_die_temp_stop,
 			       SHELL_CMD(set, NULL,
 					 "Set die temperature threshold for stop charging",
-					 cmd_die_temp_stop_set),
+					 cmd_charger_die_temp_stop_set),
+			       SHELL_CMD(get, NULL,
+					 "Get die temperature threshold for stop charging",
+					 cmd_charger_die_temp_stop_get),
 			       SHELL_SUBCMD_SET_END);
 
-/* Creating subcommands (level 4 command) array for command "charger die_temp status". */
-SHELL_STATIC_SUBCMD_SET_CREATE(sub_die_temp_status,
-			       SHELL_CMD(get, NULL, "Get die temperature comparator status",
-					 cmd_die_temp_status_get),
+/* Creating subcommands (level 3 command) array for command "charger die_temp". */
+SHELL_STATIC_SUBCMD_SET_CREATE(sub_charger_die_temp,
+			       SHELL_CMD(resume, &sub_charger_die_temp_resume,
+					 "Die temperature threshold for resume charging", NULL),
+			       SHELL_CMD(stop, &sub_charger_die_temp_stop,
+					 "Die temperature threshold for stop charging", NULL),
+			       SHELL_CMD(status, NULL, "Get die temperature comparator status",
+					 cmd_charger_die_temp_status_get),
```

Comment:
I prefer to have
npmx charger die_temp status **get** -> returns value
than
npmx charger die_temp status -> returns value

All other commands end with **set** or **get**

Reply:
Reverted to the old `get`.